### PR TITLE
add org.kde.klettres

### PR DESCRIPTION
**Pull request by cho2**

### CODE REVIEW

**Comment by cho2** [comment]
bot, build org.kde.klettres

**Comment by cho2** [comment]
bot, build org.kde.klettres

**Comment by cho2** [comment]
cc @tsdgeos 
see also https://invent.kde.org/education/klettres/-/merge_requests/3

File: org.kde.klettres.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "id": "org.kde.klettres",
+    "branch": "master",
```
Comment (nedrichards) [inline org.kde.klettres.json:3]: remove this line - for flathub it must be stable and will automatically set as that by the system.

**Comment by cho2** [comment]
bot, build org.kde.klettres

**Comment by tsdgeos** [comment]
Please include         "--device=dri" in finish-args so that we don't get this warning on starting

tsdgeos@xps:~:$ flatpak run org.kde.klettres 
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open /dev/dri/card0: El fitxer o directori no existeix
libGL error: failed to load driver: i965

**Comment by tsdgeos** [comment]
Is it me or sounds do not work?

**Comment by cho2** [comment]
bot, build org.kde.klettres

**Comment by cho2** [comment]
> Please include "--device=dri" in finish-args so that we don't get this warning on starting

https://github.com/flathub/flathub/pull/1805/commits/ae228fe46bde3a430e23a546b9e2f71d26ad46d0

> Is it me or sounds do not work?

https://github.com/flathub/flathub/pull/1805/commits/13cd0b32af11563ef59a6cf37d5f5dbb30311591

File: org.kde.klettres.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "klettres",
+    "rename-icon": "klettres",
+    "finish-args": [
+        "--share=network", 
+        "--share=ipc", 
+        "--socket=x11", 
+        "--socket=wayland",
+	"--device=dri",
```
Comment (nedrichards) [inline org.kde.klettres.json:13]: nitpick, the indenting is off here now.

**Comment by tsdgeos** [comment]
Seems reasonable to me :)

File: org.kde.klettres.json (hunk 2 above)
Comment (cho2) [reply org.kde.klettres.json]: ooops sorry, fix on https://github.com/flathub/flathub/pull/1805/commits/67ce54faa5abc79f5fe278042d85d871e15107ed

**Comment by cho2** [comment]
bot, build org.kde.klettres

**Comment by cho2** [comment]
LGTM

Cuplikan layar dari 2020-09-07 19-38-07

**Comment by nedrichards** [comment]
I'll add the KDE Flathub team manually after merging.

**Pull request closed without merging**